### ISSUE TITLE
Out-of-box error when using the client browser script

### ISSUE THREAD

**Issue by brianjenkins94** [handsontable/hyperformula#594]
### Description

Out-of-box error when using the client browser script:

```
hyperformula.min.js:54 Uncaught TypeError: Cannot read property 'createToken' of undefined
    at Object.<anonymous> (hyperformula.min.js:54)
    at __webpack_require__ (hyperformula.min.js:35)
    at Object.<anonymous> (hyperformula.min.js:46)
    at __webpack_require__ (hyperformula.min.js:35)
    at Object.<anonymous> (hyperformula.min.js:46)
    at __webpack_require__ (hyperformula.min.js:35)
    at Object.<anonymous> (hyperformula.min.js:46)
    at __webpack_require__ (hyperformula.min.js:35)
    at Object.<anonymous> (hyperformula.min.js:35)
    at __webpack_require__ (hyperformula.min.js:35)
```

### Steps to reproduce

```html
<!DOCTYPE html>

<html lang="en">
<head>
	<!-- Metadata -->
	<meta charset="utf-8" />
	<meta name="viewport" content="width=device-width, initial-scale=1" />
	<meta name="description" content="" />

	<!-- Title -->
	<title>Sample That Reproduces The Issue</title>

</head>
<body>

	<!-- Libraries -->
	<script src="https://cdn.jsdelivr.net/npm/hyperformula/dist/hyperformula.min.js"></script>

	<!-- Inline Script -->
	<script>

		document.addEventListener("DOMContentLoaded", function(event) {
			const table = [];

			const hfInstance = HyperFormula.buildFromArray(table, { "licenseKey": "agpl-v3" });
		});

	</script>

</body>
</html>
```

### Your environment

* HyperFormula version: v0.4.0
* Browser Name and version: Google Chrome v87.0.4280.141
* Operating System: Windows 10 v1809

**Comment by wojciechczerniak** [comment]
Hi @brianjenkins94!

Thanks for reporting. You have to use `full.min.js` bundle:

```html
<!-- Libraries -->
<script src="https://cdn.jsdelivr.net/npm/hyperformula/dist/hyperformula.full.min.js"></script>
```

 or load the dependencies yourself:

```html
<!-- Libraries -->
<script src="https://cdn.jsdelivr.net/npm/chevrotain@7.1.0/lib/chevrotain.js"></script>
<script src="https://cdn.jsdelivr.net/npm/hyperformula/dist/hyperformula.min.js"></script>
```

I believe this is our mistake, https://handsontable.github.io/hyperformula/guide/client-side-installation.html#use-cdn is either missing deps or should use `full.min.js` bundle. IMO both. CC @scarletfog

**Comment by wojciechczerniak** [comment]
Fixed in #612, will be updated on production with the release of v1.0
